### PR TITLE
Add release notes for v2.11.1

### PR DESCRIPTION
**Pull request by ansjcy**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
Add release notes for v2.11.1

**Describe the solution you are proposing**

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] Backport Labels added.
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by praveensameneni — APPROVED** [review]

File: release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch 2.11.1
+
+### Infrastructure
+* Set Autopublish to true in Jenkins publish for performance-analyzer-commons repo [#45](https://github.com/opensearch-project/performance-analyzer-commons/pull/45)
```
Comment (gaiksaya) [inline release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md:6]: This does not concern PA in anyway right? I believe we can remove it? More related to PA-commons repo

File: release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* Upgrade performance-analyzer-commons version to 1.2.0 [#598](https://github.com/opensearch-project/performance-analyzer/pull/598)
+
+
+### Maintenance
+* Update build.gradle to use isSnapshot logic [#521](https://github.com/opensearch-project/performance-analyzer-rca/pull/521)
+* Add separate metric for cluster manager service events and metrics [#579](https://github.com/opensearch-project/performance-analyzer/pull/579)
+* Add separate metric in commons repo for cluster manager service events and metrics [#51](https://github.com/opensearch-project/performance-analyzer-commons/pull/51)
```
Comment (gaiksaya) [inline release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md:13]: Same as above

File: release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md (hunk 1 above)
Comment (ansjcy) [reply release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md:6]: We don't have dedicated release notes for rca and commons repo, they are all included in the release notes in PA. You can check previous release notes for more information.

File: release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md (hunk 2 above)
Comment (ansjcy) [reply release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md:13]: Same as above

File: release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md (hunk 1 above)
Comment (gaiksaya) [reply release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md:6]: You can make the call as maintainers but PA-commons follows is an independent component with its own release cycle and versioning. Would recommend to add release notes for that component so when users go and check what was new in PA-commons 1.2.0 that was used in this release, it should be there. For example: common-utils new features wont be included in dependent component release notes, etc.

**Review by getsaurabh02 — APPROVED** [review]

**Pull request merged**